src/core/isTruthy.spec.js

```diff
@@ -0,0 +1,26 @@
+const test = require('tape')
+
+const isFunction = require('./isFunction')
+
+const identity = x => x
+
+const isTruthy = require('./isTruthy')
+
+test('isTruthy core', t => {
+  t.ok(isFunction(isTruthy))
+
+  t.equal(isTruthy(undefined), false, 'returns false for undefined')
+  t.equal(isTruthy(null), false, 'returns false for null')
```

Comment:
I didn't do NaN or Infinity because they were numbers but happy to add them in to all.

I think I'm learning that if I think something might be a good idea I should just do it because you're going to suggest it :)